pkg/morph/client/container/client.go

```diff
@@ -60,10 +60,6 @@ func NewFromMorph(cli *client.Client, contract util.Uint160, fee fixedn.Fixed8,
 		o.staticOpts = append(o.staticOpts, client.WithCustomFee(putNamedMethod, o.feePutNamed))
 	}
 
-	if !o.disableNotarySigning {
-		o.staticOpts = append(o.staticOpts, client.TryNotary())
```

Comment:
But this changes the default behavior, so it's not just `drop ...`

Reply:
ok, chagned the message, check it out

Reply:
changed one more time. let defaults be changed in the first commit while the second can be still "Drop"